.github/workflows/test-build.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         - trixie
         - bookworm
         - bullseye
+        - buster
```

Comment:
Is there any reason not to squash this into commit 14b94d049c9c7c545754c2d6b5709cd218891c55?

Reply:
Was pondering if we want to keep it. I'll squash it.

Reply:
squashed